src/Admin.php

```diff
@@ -71,12 +71,12 @@ public static function init() {
    * @see http://www.paulund.co.uk/hide-wordpress-update-notice
    * @see http://premium.wpmudev.org/blog/hide-the-wordpress-update-notification/
    */
-  public static function removeUselessCoreUpdateNagMessages() {
-    if (!current_user_can('update_core')) {
+  public static function removeNagNotices() {
       // @see wp-admin/includes/admin-filters.php
       remove_action('admin_notices', 'update_nag', 3);
       remove_action('admin_notices', 'maintenance_nag', 10);
-    }
+      //removes all nag notices including from woocommerce
+      remove_all_actions('admin_notices');
```

Comment:
We cannot remove all admin notices, because this would remove any kind of feedback for the user's operations, including error messages and success/completion messages.